src/features/ui/components/Button/index.ts

```diff
@@ -6,7 +6,38 @@ import { StyleReset } from '../StyleReset'
 
 type ButtonProps = {
   size?: 'small' | 'medium'
-  accent?: 'normal' | 'primary' | 'destructive' | 'edit'
+  accent?: 'normal' | 'primary' | 'destructive' | 'silent'
```

Comment:
"Edit" is related to a use case of the events domain: instead, we try to find an alternative way to describe either intent, or presentation – which is safe for generic use in button.